org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/customservice/synapse/utils/Utils.java

```diff
@@ -1359,22 +1359,30 @@ public static String sanitizeTag(String tag) {
 
     public static void downloadConnector(String groupId, String artifactId, String version, File targetDirectory)
```

Comment:
Shall we modify the same method and the update the usage of it as we only have one or two use cases currently